public/preScript.js

```diff
@@ -3,16 +3,22 @@ window.websocket.onopen = function() {
     console.log("Websocket opened");
 }
 
-if (window.location.href.toString().indexOf("localhost") !== -1) {
+if (window.location.href.toString().indexOf("localhost") !== -1 && false) {
```

Comment:
you really like this "&& false", do you?
Pls remove

Reply:
the serviceworker is needed for push notifications, it should not be disabled on localhost like that

Reply:
To Bypass the serviceworker cache check "DeveloperTools->Application->Service Workers->Bypass for network"